schemas/code_signature.yml

```
@@ -79,3 +79,22 @@
         This is used to identify the application manufactured by a software vendor.
         The field is relevant to Apple *OS only.
       example: com.apple.xpc.proxy
+
+    - name: digest_algorithm
+      level: extended
+      type: keyword
+      short: Hashing algorithm used to sign the process.
+      description: >
+        The hashing algorithm used to sign the process.
+
+        This value can distinguish signatures when a file is signed multiple times
+        by the same signer but with a different digest algorithm.
+      example: sha256
+
+    - name: timestamp
+      level: extended
+      type: date
+      short: When the signature was generated and signed.
+      description: >
+        Timestamp of when the code signature was generated and signed.
```

"Timestamp of when" feels a little off to me, may be this?
```suggestion
        Date and time when the code signature was generated and signed.
```

---

This sound right @gabriellandau?

---

Sure 👍

---

👍